integration-tests/contract-lambda-view.spec.ts

```diff
@@ -47,6 +47,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
+      console.log("lambda 1 contract : "+contract.address)
```

Comment:
console.log could be removed in this test file

Reply:
removed